### ISSUE TITLE
bug: Non-UTF8 Character Encoding is not handled properly

### ISSUE THREAD

**Issue by wbh1** [wbh1/grafana-sqlite-to-postgres#34]
Various issues have been opened to report issues with importing databases to Postgres where the error is always something along the lines of:
```console
invalid byte sequence for encoding "UTF8"
```

This issue is to combine all of those issues into a centralized place for tracking.

### Linked issues:
- #19 
- #28 

### Resources:
- https://polyglot.jamie.ly/programming/2019/07/01/grafana-sqlite-to-postgres.html
- [pgloader](https://github.com/dimitri/pgloader)

---
# Reporting your issue
If you experience this issue, please leave a comment below containing your error output. Additionally, please run a command to show the encoding of your sqlite database and include it, like so:
```console
$ file /tmp/grafana.db
/tmp/grafana.db: SQLite 3.x database, last written using SQLite version 3035004, file counter 463, database pages 216, cookie 0x16c, schema 4, UTF-8, version-valid-for 463
```

**Comment by wbh1** [comment]
The issue is with `BLOB` typed columns and importing them into bytea typed columns in postgres. May have to do with the `HexDecode` function I run which decodes that column into its raw data format which doesn't have a text encoding applied.

https://github.com/wbh1/grafana-sqlite-to-postgres/blob/68bad93236ecf83f19652ab2d78d86da442368b8/pkg/sqlite/sanitize.go#L61-L88

Should consider the decoding being applied during insertion like [here](https://stackoverflow.com/questions/3103242/inserting-text-string-with-hex-into-postgresql-as-a-bytea/3103416#3103416).

**Comment by wbh1** [comment]
Closed by: 76742b9bd66db00cad8ccc2e29a625c754375be8
